docs/components/hand-tracking-controls.md

```diff
@@ -25,7 +25,8 @@ Use `hand-tracking-controls` to integrate [hand tracked input][webxrhandinput] i
 | hand                 | The hand that will be tracked (i.e., right, left). | left                 |
 | modelColor          | Color of hand material.                                                                | white         |
 | modelStyle           | Mesh representing the hand or dots matching the joints        | mesh
-
+| startPinchDistance    | Maximum distance between tip of thumb and index finger before a pinch gesture is started. | 0.015 |
```

Comment:
Should we make it configurable? Is not a value that clearly works better for most cases?

Reply:
This is the best value I've found from my personal tests--happy to hardcode for that. The even better approach would use the `radius` property of the joint poses, but maybe best to wait until we have more than one example piece of hardware to determine the best number using that. If you confirm, I'll remove both configurable properties.

Reply:
The latest commit removes the properties.